### ISSUE TITLE
Podman still doesnt work with Wings despite it being supported now

### ISSUE THREAD

**Issue by b4dsk** [pterodactyl/panel#4595]
### Current Behavior

```
FATAL: [Dec  5 12:25:57.284] failed to configure docker environment error=Error response from daemon: compat network create can only support one IPAM config

Stacktrace:
Error response from daemon: compat network create can only support one IPAM config
```

Wings still gives this error even though it supports Podman now, it gave the same error with previous versions of Wings

### Expected Behavior

Wings should boot normally and work as it supports podman now

### Steps to Reproduce

1. Setup Wings 1.11
2. Install podman and podman-docker
3. Configure & start wings

### Panel Version

1.11.0

### Wings Version

1.11.0

### Games and/or Eggs Affected

All

### Docker Image

All

### Error Logs

```bash
https://ptero.co/wijecyhifo
```

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by matthewpi** [comment]
What version of Podman are you using?  Just post the output of `podman info`

**Comment by b4dsk** [comment]
```
host:
  arch: arm64
  buildahVersion: 1.23.1
  cgroupControllers:
  - cpuset
  - cpu
  - io
  - memory
  - hugetlb
  - pids
  - rdma
  - misc
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: 'conmon: /usr/bin/conmon'
    path: /usr/bin/conmon
    version: 'conmon version 2.0.25, commit: unknown'
  cpus: 4
  distribution:
    codename: jammy
    distribution: ubuntu
    version: "22.04"
  eventLogger: journald
  hostname: instance-20221129-1220
  idMappings:
    gidmap: null
    uidmap: null
  kernel: 5.15.0-1021-oracle
  linkmode: dynamic
  logDriver: journald
  memFree: 15124488192
  memTotal: 25153245184
  ociRuntime:
    name: crun
    package: 'crun: /usr/bin/crun'
    path: /usr/bin/crun
    version: |-
      crun version 0.17
      commit: 0e9229ae34caaebcb86f1fde18de3acaf18c6d9a
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
  os: linux
  remoteSocket:
    exists: true
    path: /run/podman/podman.sock
  security:
    apparmorEnabled: true
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: false
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: false
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: 'slirp4netns: /usr/bin/slirp4netns'
    version: |-
      slirp4netns version 1.0.1
      commit: 6a7b16babc95b6a3056b33fb45b74a6f62262dd4
      libslirp: 4.6.1
  swapFree: 0
  swapTotal: 0
  uptime: 147h 40m 32.89s (Approximately 6.12 days)
plugins:
  log:
  - k8s-file
  - none
  - journald
  network:
  - bridge
  - macvlan
  volume:
  - local
registries: {}
store:
  configFile: /etc/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /var/lib/containers/storage
  graphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  imageStore:
    number: 0
  runRoot: /run/containers/storage
  volumePath: /var/lib/containers/storage/volumes
version:
  APIVersion: 3.4.4
  Built: 0
  BuiltTime: Thu Jan  1 00:00:00 1970
  GitCommit: ""
  GoVersion: go1.17.3
  OsArch: linux/arm64
  Version: 3.4.4
  ```

**Comment by matthewpi** [comment]
That would do it.  The latest version of Podman is 4.3.1 and I have ran all of my testing on Fedora Silverblue (Rawhide) which ships the latest version (including release candidates).  Because this is an API compatibility issue within Podman that was resolved in a later version, I'm not going to bother trying to resolve it.  Especially since the error you are posting is due to us trying to use multiple subnets for a network.

To even further build on this, both RHEL 8.7 and RHEL 9.1 ship Podman 4.2.  While Ubuntu 22.04 ships 3.4.4 which isn't even the latest 3.4.x patch.  I doubt Podman and Ubuntu will be supported until a new release comes out or a third-party package providing a newer version is available.

Right now, Podman support should be considered experimental and as a best effort.  If you are able to reproduce any issues on a 4.2.x or 4.3.x release of Podman, I'll look into it.
